pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -876,11 +877,13 @@ private void GetMaxMzIntensity(out double maxMz, out double maxIntensity)
             }
         }
 
-        private void GetMaxMobility(out double maxIonMobility)
+        private void GetIonMobilityRange(out double minIonMobility, out double maxIonMobility)
         {
+            minIonMobility = double.MaxValue;
             maxIonMobility = 0;
             foreach (var spectrum in _msDataFileScanHelper.MsDataSpectra)
             {
+                var spectrumMinMobility = Math.Abs(spectrum.MinIonMobility ?? spectrum.IonMobility.Mobility ?? 0);

```

Comment:
Seems inconsistent to use zero here and double.MaxValue above. Maybe you should just use zero in both places?